woocommerce/wc-admin
# Add console warning on Inbox notes character limit

Fixes #7859

This PR adds a console warning to notify third party developers on the upcoming Inbox notes 320 character limit. In order to count actual text, it strips out all HTML tags using `sanitize` from `dompurify` module.

### Screenshots
![image](https://user-images.githubusercontent.com/3747241/139403319-b75cfb82-91e7-4406-9958-1dfb851a0398.png)

### Detailed test instructions:

- Open browser console, ensure `Warning` logging level is enabled
- Navigate to WooCommerce > Home
- In developer console, observe the warning as per screenshot attached as one of remote inbox notes is currently exceeding the 320 character limit.
- Optionally, you can create your own note by using WooCommerce Admin Test Helper and modify its content in `wp_wc_admin_notes` table to quickly test multiple offending notes.

No changelog
